website/docs/r/schema.html.markdown

```diff
@@ -40,23 +43,26 @@ resource "mso_schema" "schema1" {
 
 ```hcl
 
-resource "mso_schema" "schema1" {
-  name          = "nkp12"
-  template_name = "template1"
-  tenant_id     = mso_tenant.tenant1.id
+resource "mso_schema" "demo_schema" {
+  name          = "demo_schema"
+  template_name = "Template1"
+  tenant_id     = mso_tenant.demo_tenant.id
```

Comment:
These 2 arguments are deprecated right shouldn't the example be like below?
resource "mso_schema" "demo_schema" {
  name          = "Schema3"
  name          = "demo_schema"
  template {
    name          = "Template1"
    display_name  = "TEMP1"
    tenant_id     = "623316531d0000abdd50343a"
    template_type = "aci_multi_site"
  }
  }

Reply:
I am not sure that I understand what you mean? Is the non deprecated template block example not shown in line 16-40?

Reply:
as discussed removed deprecated examples from docs.